src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -39,6 +48,41 @@ class HttpGoogleCloudStorageDAO(clientSecretsJson: String, dataStoreFactory: Dat
      .build()
   }
 
+  override def createGoogleGroup(userId: String, accessLevel: String, workspaceName: WorkspaceName, bucketName: String) = {
+    val credential = new GoogleCredential.Builder()
+      .setTransport(httpTransport)
+      .setJsonFactory(jsonFactory)
+      .setServiceAccountId(clientSecrets.getDetails.get("client_email").toString)
+      .setServiceAccountScopes(directoryScopes)
+      .setServiceAccountUser(clientSecrets.getDetails.get("sub_email").toString)
+      .setServiceAccountPrivateKeyFromP12File(new java.io.File("/test.broad.p12")) //add this file to jenkins?
+      .build()
+
+    val directory = new Directory.Builder(httpTransport, jsonFactory, credential).setApplicationName("firecloud:rawls").build()
+    val group = new Group().setEmail(s"FC-${workspaceName.namespace}_${workspaceName.name}-${accessLevel}@test.broadinstitute.com").setName(s"FireCloud ${workspaceName.namespace}/${workspaceName.name} ${accessLevel}")
+    val member = new Member().setEmail(userId).setRole(GCSAccessLevel.toGoogleString(GCSAccessLevel.Owner))
+
+    directory.groups().insert(group).execute()
+    directory.members().insert(s"FC-${workspaceName.namespace}_${workspaceName.name}-${accessLevel}@test.broadinstitute.com", member).execute()
+  }
+
+  override def setGroupACL(userId: String, groupId: String, bucketName: String, groupRole: String): Unit = {
+    val credentialx = new GoogleCredential.Builder()
+      .setTransport(httpTransport)
+      .setJsonFactory(jsonFactory)
+      .setServiceAccountId(clientSecrets.getDetails.get("client_email").toString)
+      .setServiceAccountScopes(directoryScopes)
+      .setServiceAccountUser(clientSecrets.getDetails.get("sub_email").toString)
+      .setServiceAccountPrivateKeyFromP12File(new java.io.File("/test.broad.p12")) //add this file to jenkins?
+      .build()
+
+    val credential = getCredential(userId)
+
+    val storage = new Storage.Builder(httpTransport, jsonFactory, credential).setApplicationName("firecloud:rawls").build()
+    val acl = new com.google.api.services.storage.model.BucketAccessControl().setEntity("group-" + groupId).setRole(groupRole).setBucket(bucketName)
```

Comment:
Can you import this a bit harder so `you.don't.have.to.reference.the.entire.Thing()`?